packages/KPagination/KPagination.spec.js

```diff
@@ -0,0 +1,10 @@
+import { mount } from '@vue/test-utils'
+import KPagination from '@/KPagination/KPagination'
+
+describe('KPagination', () => {
```

Comment:
I think we need to add some tests to this. Can you add tests that check things are rendered correctly with different props passed?